### PR TITLE
Tab bar icon padding + report font

### LINKED ISSUE #4: [Tab bar] icons padding; [Report] title font stylin
Like the design, looks fancy, chill, and restrained at the same time!

Maybe more spacing/padding above the icons in bottom tab bar? Also talked about title for no reports should look like journal message too, could try and do both maybe? How would you feel about a PR for this?

Relevant links
https://reactnavigation.org/docs/tab-based-navigation.html
https://developer.android.com/training/implementing-navigation/lateral.html
https://developer.apple.com/documentation/uikit/uitabbarcontroller
(**EDIT:** fixed dupe iOS link for intended Android counterpart)

iPhone X screenshot:
nothing-to-report-yeti

### PR DESCRIPTION
**Pull request by rdela**
closes #4 

Have iPhone X + sim screenshots I can add if anyone wants to see them!

### CODE REVIEW

File: screens/Reports.js [hunk 1]
```diff
@@ -115,7 +117,7 @@ export default class Reports extends Component<*, State> {
         <StatusBar barStyle="light-content" backgroundColor={OLD_GERANIUM} />
         <BackgroundGradient />
         <ChartsContainer
-          contentContainerStyle={{ alignItems: 'center', paddingVertical: 36 }}
+          contentContainerStyle={{ alignItems: 'center', paddingVertical: 48 }}
```
Comment (rdela) [inline screens/Reports.js:122]: I changed value https://github.com/kkemple/hwyd/blob/f46d25747ed57e921d3c49e51b43af2a2c597484/screens/Reports.js#L99 from `7` to `2` during dev so I could check out graph, which led to this `paddingVertical` increase from `36` to `48`

**Comment by kkemple** [comment]
could you add screen shots?

**Comment by rdela** [comment]
## Tab Bar Padding

### Before (Today screen):

img_6170-20180318-174840

### After (Today screen):

img_6172-20180318-174903

**Comment by rdela** [comment]
## Report screen font 

### Before:

nothing to report yet before

### After:

nothing to report yet after

**Comment by rdela** [comment]
## Report graph `paddingVertical`

### Before from sim w/ inspector up

screen shot 2018-03-18 at 135038

### After from sim w/ inspector up

screen shot 2018-03-18 at 135231

### Before iPhone X

img_6174-20180318-180559

### After iPhone X

img_6175-20180318-180604

**Review by SaraVieira — APPROVED** [review]
Looks great!!

**Comment by kkemple** [comment]
this looks amazing! ❤️

only feedback is can we center title vertically for no reports state, like we do on journal page?

**Comment by kkemple** [comment]
or should we move journal one up? 🤔

**Comment by rdela** [comment]
>  center title vertically for no reports state, like we do on journal page?

Gonna try this one first since it was your 1st intuition, got grabs of current empty and with 1 journal entry for reference in case we change our minds and switch it up later 📸

**Comment by rdela** [comment]
nü sqreenshawtz! 🤘 

## empty Reports state
img_6203-20180320-162209

## Reports with `checkIns.length` check set to `2` to preview graph (unchanged)
img_6204-20180320-162218

**Comment by kkemple** [comment]
amazing!! thanks so much for this! :heart:

**Review by kkemple — APPROVED** [review]

**Comment by kkemple** [comment]
also, i've added you as a collaborator so you can check out this repo now unless you want to continue to work from your fork, up to you!

**Pull request merged**